asserts/account_key.go

```diff
@@ -96,6 +104,21 @@ func (ak *AccountKey) checkConsistency(db RODatabase, acck *AccountKey) error {
 	if err != nil {
 		return err
 	}
+	// XXX: Make this unconditional once account-key assertions are required to have a name.
+	if ak.Name() != "" {
+		assertions, err := db.FindMany(AccountKeyType, map[string]string{
```

Comment:
Agreed right back. Maybe a comment here would be a good idea? It might help someone debug if it ever does happen.